dashboards/resources/cluster.libsonnet

```diff
@@ -140,7 +140,7 @@ local template = grafana.template;
          })
         .addPanel(
           g.panel('CPU Utilisation') +
-          g.statPanel('1 - avg(rate(node_cpu_seconds_total{mode="idle", %(clusterLabel)s="$cluster"}[%(grafanaIntervalVar)s]))' % $._config) +
+          g.statPanel('1 - avg(rate(node_cpu_seconds_total{mode=~"idle|iowait|steal", %(clusterLabel)s="$cluster"}[%(grafanaIntervalVar)s]))' % $._config) +
```

Comment:
@julianwiedmann @paulfantom   this isn't right. the `avg` function will average  over all three idle, iowait and steal modes, which will lower the right hand side value in (1 - v), thereby reporting higher utilization that the actual.

`(1 - (avg(rate(node_cpu_seconds_total{mode=~"idle") + avg(rate(node_cpu_seconds_total{mode=~"iowait") + avg(rate(node_cpu_seconds_total{mode=~"steal")))`  might be right ?

Reply:
Great catch! Your proposal sounds good to me, but maybe let's simplify this to `1 - sum(avg by (mode) (rate(node_cpu_seconds_total{mode=~"idle|iowait|steal"}[5m])))`? Could you create a PR?